ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -46,11 +46,7 @@ public override async Task<IActionResult> OnGetAsync(int id, bool isBeingSubmitt
             }
 
             ClinicalDetails = Notification.ClinicalDetails;
-            if (ClinicalDetails == null) {
```

Comment:
I removed this everywhere as we always create this when a notification gets created, so it's unnecessary